drivers/ethernet/eth_mcux.c

```diff
@@ -66,6 +72,11 @@ struct eth_context {
 	 */
 	struct net_if *iface;
 	enet_handle_t enet_handle;
+#if defined(CONFIG_PTP_CLOCK_MCUX)
+	struct device *ptp_clock;
+	enet_ptp_config_t ptp_config;
+	float clk_ratio;
```

Comment:
Is PTP in general/other drivers depend on float to provide PTP support. I'd say, it's a bit strange to see, I wonder why fixed point wasn't used.